clang/include/clang/Sema/Sema.h

```diff
@@ -12077,7 +12077,8 @@ class Sema final {
     KernelAllocateStorage,
     KernelUseAssembly,
     KernelCallDllimportFunction,
-    KernelCallVariadicFunction
+    KernelCallVariadicFunction,
+    KernelCallDisallowedMethod
```

Comment:
```suggestion
    KernelCallUndefinedFunction
```
Since the diagnostic tells `call an undefined function w/o SYCL_EXTERNAL attribute`.